docs/interactions/structured-messages/template.md

```diff
@@ -113,3 +113,30 @@ Primary parameters are used by default, however, some parameters are unique or o
 | **`structured_content.items.title`** | Truncated to 25 characters. |
 | **`structured_content.items.payload`** | Automatically gets populated as a random hex if blank. |
 | **`structured_content.attachment_id`** | Supports private attachments. |
+
+## Example: WhatsApp (Reply Buttons)
+
+<img class="img-fluid" width="398" src="../../../img/structured-messages-template-whatsapp.png">
+
+### Properties Unique to this Channel
+
+Primary parameters are used by default, however, some parameters are unique or overwritten by parameters specific to this example.
+
+| API Property | Type | Specificity |
```

Comment:
You should also mention somewhere that url isn't supported on both the body and the items. (Yes I know you already put "reply only" in the item, but better be real clear)

Reply:
yep, added